docs/specifications/tests/Nameserver-TP/nameserver13.md

```diff
@@ -0,0 +1,87 @@
+# nameserver13: Test for EDNS truncated response
+
+### test case identifier
+
+**Nameserver13**
+
+### objective
+
+EDNS is a mechanism to announce capabilities of a DNS implementation,
+and is now basically required by any new functionality in DNS such as
+DNSSEC ([rfc 6891]).
+
+[RFC 6891, section 7] states that an OPT record is supposed to be included
+in a truncated response, where-in in the request, an OPT pseudo-RR is present.
+
+In this test case, we will query with DO bit set to '1' and forcing a
+truncated response with setting buffer size to 512 in the query. We will also
+make sure that the resolution is not retried using TCP.
```

Comment:
1. I do not think we should write "we will".
2. How do you make the response being truncated?
3. Retrying with TCP is not done automatically.

Reply:
It seems the fallback is automatically to TCP

Reply:
Text modified. Please check.